error_reporting/tests/system.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import unittest
+
+from google.cloud import error_reporting
+from google.cloud.gapic.errorreporting.v1beta1 import (
+    error_stats_service_client)
+from google.cloud.proto.devtools.clouderrorreporting.v1beta1 import (
+    error_stats_service_pb2)
+from google.protobuf.duration_pb2 import Duration
+
+from test_utils.retry import RetryResult
+
+
+class _ErrorStatsGaxApi(object):
```

Comment:
Switched it to a function. I'm hesitant to add it to the module because it's not really a core use of the API, just a helper for a system test, and so I don't want to bloat the module. As GAPIC docs improve people can just rely on the GAPIC itself it they really want to do this stuff.